### PR TITLE
change docs to reference downloading of zip files instead of .img

### PR DESCRIPTION
**Pull request by shaunmulligan**

### CODE REVIEW

**Comment by shaunmulligan** [comment]
@dfunckt

File: pages/deployment/wifi.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ In general the network config follows the [ConnMan][connman] configuration file
 
 ## Changing your Network Configuration
 
-On all devices excluding the [Intel Edison](/installing/gettingStarted-Edison), it is possible to change your wifi SSID or Passphrase after downloading the `.img`.
+It is possible to change your wifi SSID or Passphrase after downloading the `.zip`, however it is not possible to change the wifi SSID and passphrase of devices that run from internal eMMC without reflashing the device.
```
Comment (dfunckt) [inline pages/deployment/wifi.md]: I would just say "downloading the image". Also, the word "passphrase" is capitalised once, and then all lower-case.

File: pages/installing/gettingStarted-Humming.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ application. A dialog will appear prompting you to specify how your device conne
 
 While the file downloads, ensure your SD card is formatted in [FAT32][fat32] ([WikiHow][wikihow] has [instructions][wikihow_format] on how to do this).
 
-Once the download is finished you should have a `.img` file with a name like `resin-myFleet-0.1.0-0.0.14.img` where myFleet is the name you gave your application on the dashboard.
+Once the download is finished you should have a `.zip` file with a name like `resin-myFleet-0.1.0-0.0.14.zip` where myFleet is the name you gave your application on the dashboard. Extract the `.zip` to a folder and you should see a OS `.img` in the folder. We will use this in the next steps.
```
Comment (dfunckt) [inline pages/installing/gettingStarted-Humming.md]: see a OS -> see an OS.

(this is in all Getting Started files)

File: shared/flashingOsToBootMedia/_default.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ For simplicity this tutorial will assume you are using [Etcher][etcher-link]. On
 
 To create a bootable resinOS {{ $device.bootMedia }} follow these 3 easy steps:
 
-1. Click "Select image" button and find your applications resinOS `.img` file.
+1. Click "Select image" button and find your applications resinOS `.zip` file. There is no need to extract it beforehand, etcher will do that for you.
```
Comment (dfunckt) [inline shared/flashingOsToBootMedia/_default.md]: etcher -> Etcher

**Comment by dfunckt** [comment]
Noted a couple of typos, but LGTM.

**Comment by shaunmulligan** [comment]
This branch is way too old now and things have changed alot, so closing.

**Pull request closed without merging**